app/views/posts/flat.haml

```diff
@@ -7,6 +7,8 @@
     = stylesheet_link_tag 'application'
     - if current_user.try(:layout).present?
```

Comment:
`= stylesheet_link_tag "layouts/#{current_user&.layout || 'default'}"` maybe?

Reply:
Would that work? I guess that would work